tests/unit/test_queries.py

```diff
@@ -24,7 +24,6 @@
     45,
     47,
     49,
-    50,
```

Comment:
This passes now too for some reason.

Reply:
Originally I was seeing https://gpuci.gpuopenanalytics.com/job/dask/job/dask-sql/job/prb/job/dask-sql-prb/2755/CUDA_VER=11.5,LINUX_VER=ubuntu18.04,PYTHON_VER=3.9,RAPIDS_VER=23.06/testReport/junit/tests.unit/test_queries/test_gpu_query_q50_sql_/ but now I'm seeing https://gpuci.gpuopenanalytics.com/job/dask/job/dask-sql/job/prb/job/dask-sql-prb/2759/CUDA_VER=11.5,LINUX_VER=ubuntu18.04,PYTHON_VER=3.9,RAPIDS_VER=23.06/testReport/junit/tests.unit/test_queries/test_gpu_query_q50_sql_/ ?